osf/caching.py

```diff
@@ -0,0 +1,37 @@
+import functools
+from django.core.cache import cache
+from django.conf import settings
+
+STORAGE_USAGE = 'storage_usage:{node_id}'
```

Comment:
It's important to not hard code the keys because certain caches (`LocMemCache`, etc.) don't allow certain chars, so here we only have to change it once.